app/src/main/java/me/phh/treble/app/Misc.kt

```diff
@@ -188,8 +188,19 @@ object Misc: EntryStartup {
                 SystemProperties.set("persist.sys.fflag.override.settings_fuse", if (!value) "true" else "false")
             }
             MiscSettings.backlightScale -> {
-                val value = sp.getBoolean(key, false)
-                SystemProperties.set("persist.sys.phh.backlight.scale", if (value) "1" else "0")
+                val value = sp.getString(key, "System Default")
```

Comment:
should be "system default"?

Reply:
system default is -1 or the change listed in rw-system
of is -1 in every case
on is the value from /sys... in every case

so system default should be right one. the name is your thing... that was only my idea...